sql/step_82_logout_menu_item.sql

```diff
@@ -0,0 +1,3 @@
+INSERT INTO OH_MENUITEM VALUES ('logout','angal.menu.btn.logout','angal.menu.logout','x','L','main','none','N',13);
+INSERT INTO OH_GROUPMENU (GM_UG_ID_A, GM_MNI_ID_A, GM_ACTIVE) VALUES ('admin','logout',1);
+INSERT INTO OH_GROUPMENU (GM_UG_ID_A, GM_MNI_ID_A, GM_ACTIVE) VALUES ('guest','logout',1);
```

Comment:
I don't know if it is really needed a MENUITEM for the logout: the button should always appear when SINGLEUSER=no, and should not appear when SINGLEUSER=yes.
Having a MENUITEM entry make sense when we want eventually disable the logout function for some user, but we do?
Of course, having this entry simplify the button construction in the main menu...